guides/plugins/plugins/storefront/customize-header-footer.md

```diff
@@ -0,0 +1,50 @@
+---
+nav:
+  title: Customize Header/Footer
+  position: 160
+
+---
+
+# Customize Header/Footer
+
+## Overview
+
+With the introduction of ESI loading for the header and footer, the way how to customize the header and footer has changed.
+E.g. it is no longer possible to customize the header and footer depending on the current page data.
```

Comment:
It feels like a common case can be something like the account menu needs to be changed when logged in (like additional navigation points in SwagCommercial). There should be a way to make use of the currently logged-in customer and change the menu as well. Any hints on how to approach that?

Reply:
if a customer is logged-in the header and footer are also not cached anymore. so it should like before

Reply:
Sad. It could've been a huge gain :/ I see why though